x-pack/plugins/cloud_security_posture/public/pages/rules/rules_counters.tsx

```diff
@@ -0,0 +1,319 @@
+      integrationLink: cspmIntegrationLink,
+      learnMoreLink: 'https://ela.st/cspm-get-started',
+    },
+    cis_k8s: {
+      integrationType: 'KSPM',
+      integrationName: 'Kubernetes',
+      resourceName: 'Clusters',
+      integrationLink: kspmIntegrationLink,
+      learnMoreLink: 'https://ela.st/kspm-get-started',
+    },
+    cis_eks: {
+      integrationType: 'KSPM',
+      integrationName: 'EKS',
+      resourceName: 'Clusters',
+      integrationLink: kspmIntegrationLink,
+      learnMoreLink: 'https://ela.st/kspm-get-started',
+    },
+  };
+
+  if (benchmarkRulesStats.score.totalFindings === 0) {
+    return (
+      <EuiEmptyPrompt
+        color="plain"
+        icon={<EuiImage size="fullWidth" src={noDataIllustration} alt="no_data_illustration" />}
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.csp.rulesPage.rulesCounterEmptyState.emptyStateTitle"
+              defaultMessage="Add {integrationResourceName} to get started"
+              values={{
+                integrationResourceName: `${
+                  benchmarkDynamicValues[benchmarkRulesStats.id].integrationName
+                }
+                  ${benchmarkDynamicValues[benchmarkRulesStats.id].resourceName}`,
+              }}
+            />
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.csp.rulesPage.rulesCounterEmptyState.emptyStateTitle"
+              defaultMessage="Add your {resourceName} in {integrationType} to begin detecing misconfigurations"
+              values={{
+                resourceName:
+                  benchmarkDynamicValues[benchmarkRulesStats.id].resourceName.toLowerCase(),
+                integrationType: benchmarkDynamicValues[benchmarkRulesStats.id].integrationType,
+              }}
+            />
+          </p>
+        }
+        actions={[
+          <EuiButton
+            color="primary"
+            fill
+            href={benchmarkDynamicValues[benchmarkRulesStats.id].integrationLink}
+          >
+            <FormattedMessage
+              id="xpack.csp.rulesPage.rulesCounterEmptyState.emptyPrimapryButtonTitle"
+              defaultMessage="Add {integrationType} integration"
+              values={{
+                integrationType: benchmarkDynamicValues[benchmarkRulesStats.id].integrationType,
+              }}
+            />
+          </EuiButton>,
+          <EuiButtonEmpty
+            color="primary"
+            href={benchmarkDynamicValues[benchmarkRulesStats.id].learnMoreLink}
+            target="_blank"
+          >
+            <FormattedMessage
+              id="xpack.csp.rulesPage.rulesCounterEmptyState.emptyLearnMoreButtonTitle"
+              defaultMessage="Learn more"
+            />
+          </EuiButtonEmpty>,
+        ]}
+        layout="horizontal"
+        paddingSize="m"
+      />
+    );
+  }
+
+  const counters = [
+    {
+      id: DASHBOARD_COUNTER_CARDS.CLUSTERS_EVALUATED,
```

Comment:
all counters here have the same id (`DASHBOARD_COUNTER_CARDS.CLUSTERS_EVALUATED`), it should be unique